### PR TITLE
Update Confluence Teleports for Escha - Zi'tah and Escha - Ru' Aun

### PR DESCRIPTION
**Pull request by jefferyrlc**
**_I affirm:_**
- [x ] that I agree to Topaz Next's [Limited Contributor License Agreement](https://github.com/topaz-next/topaz/blob/release/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x ] that I have read the [Contributing Guide](https://github.com/topaz-next/topaz/blob/release/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/topaz-next/topaz/blob/release/CODE_OF_CONDUCT.md)
- [x ] that I've _tested my code_ since the last commit in the PR, and will test after any later commits

**_Temporary_**:
- [ x] that I understand there are large refactoring efforts going on right now, that these efforts touch every single Lua script and binding, and that my pull request might get put on hold to ensure there are not any conflicts with the refactoring work

Pushes the teleport locations into globals/teleports for the Undulating Confluences in Qufim Island and Escha - Zi'tah. Also adds teleports for the Undulating Confluences in Misareaux Coast and Escha - Ru'Aun.

### CODE REVIEW

File: scripts/zones/Escha_RuAun/npcs/Undulating_Confluence.lua [hunk 1]
```diff
@@ -0,0 +1,26 @@
+-----------------------------------
+-- Area: Escha - Ru'Aun (289)
+--  NPC: Undulating Confluence
+-- !pos -0.163 -34.106 -471.971 289
+-----------------------------------
+local ID = require("scripts/zones/Escha_ZiTah/IDs")
```
Comment (Chronos77) [inline scripts/zones/Escha_RuAun/npcs/Undulating_Confluence.lua:6]: This variable is not used you can delete it

File: scripts/zones/Misareaux_Coast/npcs/Undulating_Confluence.lua [hunk 2]
```diff
@@ -0,0 +1,26 @@
+-----------------------------------
+-- Area: Misareaux Coast (25)
+--  NPC: Undulating Confluence
+-- !pos --48.908 -23.302 572.269 25
+-----------------------------------
+local ID = require("scripts/zones/Misareaux_Coast/IDs")
```
Comment (Chronos77) [inline scripts/zones/Misareaux_Coast/npcs/Undulating_Confluence.lua:6]: This variable is not used you can delete it

**Review by Chronos77 — COMMENTED** [review]
the content seems good to me, but can you squash your commits and set a good commit message please

**Comment by ghost** [comment]
@jefferyrlc If you're having issues with GitKraken, I might be able to help.  Seems a bit weird that all of the commit messages are showing like that.  Hit me up on discord if you'd like.

**Comment by zach2good** [comment]
It won't be an issue after this week, but we don't take submissions directly into `canary`, it's made up of `release` and whatever active `feature` branches we have open. I'll merge this into its own branch and rebase and do all the other git magic at the start of next week

**Pull request merged**